cypress/integration/advanced-functionalities/one/jitsi_bridge/Create_room.spec.js

```diff
@@ -0,0 +1,25 @@
+/// <reference types="cypress" />
+const oneURL = Cypress.env('webappLink').oneURL
+const email = Cypress.env('user').email
+const password = Cypress.env('user').password
+
+describe('Testing room creation in Jitsi Bridge', () => {
+  before(() => {
+    if (!window.sessionStorage.getItem('auth.refresh-token')) {
+      cy.login({ email, password, webappLink: oneURL })
+    }
+  })
+
+  context('Test for creating a room in Jitsi Bridge', () => {
+    it('should be able to create a room', () => {
+      cy.visit(oneURL)
+      // Here we close the start tour pop up
+      if (!window.sessionStorage.getItem('auth.refresh-token')) {
+        cy.get('.modal-header > :last-child()').click()
+      }
+      cy.get('[data-test-id="Jitsi Bridge"]').click({ force: true })
```

Comment:
You're missing one more test. Please, reference other files to see how we structure our tests